tests/foreman/ui/test_bookmarks.py

```diff
@@ -0,0 +1,516 @@
+    {'name': 'PuppetModule', 'controller': 'katello_puppet_modules'},
+    {'name': 'DockerTag', 'controller': 'katello_docker_tags'},
+    {'name': 'Registry', 'controller': 'registries'},
+    {'name': 'Host', 'controller': 'katello_hosts',
+        'setup': entities.Host},
+    {'name': 'ContentHost', 'controller': 'katello_systems'},
+    {'name': 'HostCollection', 'controller': 'katello_host_collections'},
+    {'name': 'Architecture', 'controller': 'katello_architectures'},
+    {'name': 'HWModel', 'controller': 'models'},
+    {'name': 'InstallationMedia', 'controller': 'media',
+        'setup': entities.Media},
+    {'name': 'OS', 'controller': 'operatingsystems'},
+    {'name': 'PTable', 'controller': 'ptables',
+        'setup': entities.PartitionTable},
+    {'name': 'ProvisioningTemplate', 'controller': 'provisioning_templates'},
+    {'name': 'HostGroup', 'controller': 'host_groups',
+        'setup': entities.HostGroup},
+    {'name': 'DiscoveryRule', 'controller': 'discovery_rules'},
+    {'name': 'GlobalParameter', 'controller': 'common_parameters',
+        'setup': entities.CommonParameter},
+    {'name': 'ConfigGroup', 'controller': 'config_groups',
+        'setup': entities.ConfigGroup},
+    {'name': 'PuppetEnv', 'controller': 'environments',
+        'setup': entities.Environment},
+    {'name': 'PuppetClass', 'controller': 'puppet_classes',
+        'setup': entities.PuppetClass},
+    {'name': 'SmartVariable', 'controller': 'smart_variables',
+        'setup': entities.SmartVariable},
+    {'name': 'SmartProxy', 'controller': 'smart_proxies'},
+    {'name': 'ComputeResource', 'controller': 'compute_resources',
+        'setup': entities.DockerComputeResource},
+    {'name': 'ComputeProfile', 'controller': 'compute_profiles'},
+    {'name': 'Subnet', 'controller': 'subnets',
+        'setup': entities.Subnet},
+    {'name': 'Domain', 'controller': 'domains',
+        'setup': entities.Domain},
+    {'name': 'Realm', 'controller': 'realms',
+        'setup': entities.Realm},
+    {'name': 'Location', 'controller': 'locations'},
+    {'name': 'Organization', 'controller': 'organizations'},
+    {'name': 'User', 'controller': 'users'},
+    {'name': 'UserGroup', 'controller': 'user_groups',
+        'setup': entities.UserGroup},
+    {'name': 'Role', 'controller': 'roles'},
+    {'name': 'SettingTestCase', 'controller': 'settings'},
+]
+
+
+def create_entity(entity):
+    """Take entity name and call nailgun methods to create a random instance
+
+    :param entity: Nailgun entity class to be processed
+    """
+
+    ng_entity = entity()
+    ng_entity.create_missing()
+    ng_entity.create()
+
+
+class BookmarkTestCase(UITestCase):
+    """Test for common Bookmark operations in UI"""
+
+    # CREATE TESTS
+    @stubbed()
+    @tier1
+    def test_positive_create_bookmark_populate_auto(self):
+        """Test creating a bookmark with auto-populating of the query
```

Comment:
Please drop `Test` from docstring.  This can read `Create a bookmark with auto-populating of the query`.

This comment applies to all test cases